src/plugins/intel_gpu/src/kernel_selector/cl_kernels/gemm_tiled_opt.cl

```diff
@@ -200,14 +200,14 @@ KERNEL(gemm_tiled_opt)(
 #endif // TRANSPOSE_INPUT0
 #if TRANSPOSE_INPUT1 == TRANSPOSE_X_LAST
     const __global INPUT1_TYPE* b_ptr = input1 + batch_offset_input1;
-    #if HAS_DYNAMIC_K_PADDING || INPUT1_HAS_PADDING
+    #if HAS_DYNAMIC_N_PADDING || INPUT1_HAS_PADDING
         const uint input1_offset = FUNC_CALL(get_input1_index)(OPTIONAL_SHAPE_INFO_TENSOR b, f, w, z, 1, tile_n_offset) - batch_offset_input1;
     #else
         const uint input1_offset = FUNC_CALL(get_input1_index)(OPTIONAL_SHAPE_INFO_TENSOR 0, 0, 0, 0, 1, 0);
     #endif
 #elif TRANSPOSE_INPUT1 == TRANSPOSE_Y_LAST
     const __global INPUT1_TYPE* b_ptr = input1 + batch_offset_input1;
-    #if HAS_DYNAMIC_K_PADDING || INPUT1_HAS_PADDING
+    #if HAS_DYNAMIC_N_PADDING || INPUT1_HAS_PADDING
```

Comment:
There is no HAS_DYNAMIC_N_PADDING in this kernel and the code is related with input1. So I think it's typo. After modifying this, my issue1 is fixed.

Reply:
Input 1 can be related for both n and k padding. 
I think it should be for both HAS_DYNAMIC_N_PADDINTG || HAS_DYNAMIC_K_PADDING || INPUT1_HAS_PADDING:
@e-ddykim is the original author of this part. Could you please comment? 
Also @kelvinchoi-intel  I'd like to suggest to check accuracy of various dynamic models by **turning off the sdpa_opt** which let the models use gemm instead of sdpa kernel.

Reply:
With updated commit, this code no more necessary.